www/templates/default/html/js/manager.js

```diff
@@ -1,3 +1,7 @@
+require([frontend_url + "templates/default/html/js/vendor/select2/js/select2.min.js"], function(select2) {
```

Comment:
Hint: You should add `'jquery'` to the require array and a function parameter for the jQuery module to avoid making the assumption that the dependency `WDN.jQuery` exists.